rel/files/riak_cs.schema

```diff
@@ -181,7 +180,14 @@
 %% Riak nodes prior to version 1.4.0.
 {mapping, "gc.paginated_indexes", "riak_cs.gc_paginated_indexes", [
   {default, on},
-  {datatype, flag}
+  {datatype, flag},
+  hidden
+]}.
+
+{mapping, "gc.max_workers", "riak_cs.gc_max_workers", [
+  {default, 5},
```

Comment:
How about changing default value? Five workers can make rather large load. I want to recommend 2 to begin with.